### DISCUSSION TITLE
`readonly()` attribute should NOT disable the field

### DISCUSSION THREAD

**Discussion by msucevan** [laravel/nova-issues#6483] [Ideas & Feature Requests] [answered]
Readonly attribute must to add the `readonly` attribute. Right now the function readonly() add `readonly` and `disabled`.

This is not the intention of `readonly` attribute, the `readonly` attribute should not make writeable the field, but the content must be appended to the payload. This makes the readonly() function completely useless.

**Comment by crynobone** [comment, marked as answer]
Released with Laravel Nova 5.0.0

https://nova.laravel.com/docs/v5/releases#new-field-immutable-method

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
